ci/archs.sh

```diff
@@ -52,18 +52,21 @@ declare -A ARCHS_GO=(
     [aarch64]=arm64
 )
 
-# Key is one of ARCHS
-declare -A ARCHS_RUST=(
-    [i386]=i686
-    [amd64]=x86_64
-    [armel]=armv5_eabi
-    [armhf]=armv7_eabihf
-    [aarch64]=aarch64
+# for .so files comming from libtelio, libdrop and libmoose
+declare -A ARCHS_SO_REVERSE=(
```

Comment:
So this is how the libtelio, libdrop and libmoose produce artifacts in their releases:
| lib | arch |
| --- | --- |
| libdrop: | aarch64, armv5     , armv7, i686, x86_64 |
| libmooose-nordvpnapp: | aarch64, armv5_eabi, armv7_eabihf, i686, x86_64 |
| libmooose-worker: | aarch64, armv5_eabi, armv7_eabihf, i686, x86_64 |
| libtelio: | aarch64, armv5, armv7hf, i686, x86_64 |

So this dict is just a union of all the architectures from the libs, and it's mapped to what golang expects in cgo bindings. 

(I restored `ARCHS_RUST` because I'll need it later but, currently it's not used anywhere. At least I didn't find any use of it either in nordvpn-linux or nordvpn-linux-build :thinking: )

Did I answer you question? :smile:

Reply:
this will be clarified later if libraries have only different names or also different arch.